### ISSUE TITLE
KeyError: 'consumer_key' on Linux with old and latest python installed, on windows all fine.

### ISSUE THREAD

**Issue by marevix338** [upwork/python-upwork#60]
Traceback (most recent call last):
  File "best-upwork-script.py", line 15, in <module>
    config = upwork.Config(
  File "/usr/local/lib/python3.8/dist-packages/upwork/config.py", line 22, in __init__
    config["consumer_key"],
KeyError: 'consumer_key'

so by this i cant bring up my script to work on linux, used 3.8 python to 3.10.5 python from 3rd party repos
no luck, but on windows with 3.10.4 and 3.10.5 latest stable all works like a charm
`

upwork config & token
```
from upwork.routers.jobs import search
import requests
import upwork
import time


token = {
    "access_token": "xxxxxxxxxxxxxxxxx",
    "expires_at": xxxxxxxx,
    "expires_in": xxxxxxx,
    "refresh_token": "xxxxxxxxxxxxxxxxxxxxxx",
    "token_type": "Bearer",
}
#
config = upwork.Config(
    {
        "client_id": "xxxxxxxxxxxxxxxxxxxxxxxx",
        "client_secret": "xxxxxxxxxxxxxxxxxxxxxxxxxxxx",
        "token": token
    }
)
client = upwork.Client(config)
variable = search.Api(client).find(params)
print(variable)
```
nothing major have changed from 3.8 version

`/usr/local/lib/python3.8/dist-packages/upwork/config.py`

**Comment by marevix338** [comment]
upd. 
```
3.10.5 latest stable from windwos app store 
-same Error 
tried latest pypy3 3.9 
-same error
but 3.10.4 works
```

**Comment by marevix338** [comment]
upd. 3.10.4 from sources

`Traceback (most recent call last):
  File "script-of-all-scripts.py", line 15, in <module>
    config = upwork.Config(
  File "/somepath/.local/lib/python3.10/site-packages/upwork/config.py", line 22, in __init__
    config["consumer_key"],
KeyError: 'consumer_key'`

but on windows 3.10.4 version runs after re installing python/module
and not works on 3.10.5

**Comment by marevix338** [comment]
upd. resolved by using only : `python-upwork-oauth2==3.0.4`
so u can't have both like this:
```
python-upwork==2.1.0
python-upwork-oauth2==3.0.4
```
use `python -m pip install python-upwork-oauth2`
so u will be shure u installing module for interpreter u are currently using.
[https://github.com/upwork/python-upwork-oauth2](url)
others are outdated  and should be removed i guess, so ppl wont be frustrated.
